part-administration.tex

```diff
@@ -947,32 +947,72 @@ \chapter{Software updates}
 \section{Introduction}
 \label{sec-updates-introduction}
 
-This chapter describes patch release updates. Upgrades from older versions (e.g. 1.1 or 1.2)
-are covered in \appref{app-migration}. The same is true for upgrades from SQL Ledger 2.6 or 2.8.
+This chapter describes patch and minor version release updates from 1.4 or higher.
+Upgrades from older versions (e.g. 1.2 or 1.3) are covered in \appref{app-migration}.
+The same is true for upgrades from SQL Ledger 2.6 or 2.8.
 
-The remainder of the chapter discusses the steps to be executed. Please note that the instructions
-below mean a window where the application can't be used during the entire execution of the procedure.
+The remainder of this chapter discusses the steps to be executed. Please note that during execution of the instructions below the application isn't to be used.
 
 \section{Backups}
 \label{sec-updates-backups}
 
+All upgrades should start by creating backups of the system as it runs.  This means:
+
 \begin{itemize}
 \item Backup database roles (authorizations)
 \item Backup database content (structure and data)
 \item Backup software and settings files
 \end{itemize}
 
+The database backups can be created through the \texttt{setup.pl} web interface.  The
+database content backup can also be created using the \texttt{ledgersmb-admin} administration
+command line tool.
 
+A backup of the software itself is only relevant when using a source installation. When using
+the (recommended) Docker installation, this step can be skipped; it \textit{is} important to
+record which image version the container is based on. For source installs the backup can simply
+be performed by recursively copying the containing directory: 
+\texttt{cp -Rp /srv/ledgersmb /srv/ledgersmb.bak}
 
 \section{Software upgrade}
 \label{sec-updates-software-upgrades}
 
-@@@ Untar / use package manager
+When using Docker, the upgrade is as simple as spinning up a container with the new version.
+For source installation the procedure is simple as well: The tarball needs to be unpacked,
+replacing the existing sources.  From the archive directory, the file \texttt{ledgersmb.yaml}
+(or \texttt{ledgersmb.conf}) must be copied.  The same needs to be done for any custom CSS
+theme files in the \texttt{UI/css} directory.
+
+If your setup is using the deprecated \texttt{ledgersmb.conf} INI-style configuration
+file, you should use the migration utility \texttt{utils/migration/convert-ini-to-di}
+to generate the new YAML-based configuration format \texttt{ledgersmb.yaml}.
 
 \section{Database upgrade}
 \label{sec-updates-database-upgrades}
 
-@@@ go to setup.pl and ``Rebuild''
+After upgrading the installed software, all LedgerSMB company databases need to be
+upgraded in order to become compatible with the new software.  This upgrade is
+performed by logging into the database through \texttt{setup.pl}.  After successful
+login, the question "LedgerSMB X.YY db found. Rebuild/Upgrade?" with a \texttt{Yes}

```

Comment:
It's not clear here what happens in the case of an upgrade from 1.11.5 to 1.11.6.  Is it project policy that this type of patch upgrades will never need a database update?  If so, it should be clarified here.

Reply:
Added.